test/test_pexcept_hooks.c

```diff
@@ -0,0 +1,66 @@
+#include "unity.h"
+#include "putils/pexcept.h"
+
+PEXCEPT_T uncaught_exception;
```

Comment:
Might be a good idea to set an initial default value to this two global variables 

```c
PEXCEPT_T uncaught_exception = PEXCEPT_NONE;
enum statuses status = NOTHING_HAPPENED;
```